### ISSUE TITLE
Screen reader announces redundant/unnecessary link information in Enable Bypass Approvals dialog:A11y_VisualStudioCode_Permissions levels in Copilot CLI_ScreenReader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#321416]
## GitHub Tags
#A11yTCS;#Visual Studio Code Client; #SH_Integrated browser features_Win32_JUNE2026;#DesktopApp;#FTP;#Win32;#A11ySev3;#A11yMAS;#WCAG1.3.1;  #ScreenReader; #NVDA;

## Issue Subcategory 
Supplementary information is not programmatically associated 

## User Experience 
Screen reader users may not hear important helper text, requirements, or error context associated with a form field, so they may not understand how to complete it correctly.

## Repro Steps 
Turn on NVDA screen reader > Open VS Code > Copilot chat > Permission Picker combo box > Bypass approvals > “chat.permissions.default" link > Observe the issue: Screen reader announces redundant/unnecessary link information in Enable Bypass Approvals dialog.

## Actual Result 
After announcing the link name and role (e.g., “chat.permissions.default, link”), the screen reader further announces unnecessary information such as internal command text (e.g., “command:workbench.action.openSettings…”), which is not meaningful to users.

## Expected Result 
Screen reader should announce only meaningful and user-relevant information (link name and role). Internal or technical details (e.g., command paths or IDs) should not be exposed to assistive technologies.

## How to Fix 
 Use aria-describedby to reference visible helper or error text, or use aria-description when appropriate for concise descriptive text that is not otherwise visible. 

## Environment details 
Application Name: VS Code 
VS Code version: 1.124.2
Edition Windows 11 Enterprise
Version 25H2
OS build 26200.8390
NVDA: 2025.3.3

## Note: 

- Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
- "[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachment
https://github.com/user-attachments/assets/a20b4afa-8bc7-483a-8b95-7fdf92907ee9
